src/tools/info/sys_info.c

```diff
@@ -31,6 +31,7 @@ static const char *cpu_model_names[] = {
     [UCS_CPU_MODEL_INTEL_HASWELL]      = "Haswell",
     [UCS_CPU_MODEL_INTEL_BROADWELL]    = "Broadwell",
     [UCS_CPU_MODEL_INTEL_SKYLAKE]      = "Skylake",
+    [UCS_CPU_MODEL_INTEL_ICELAKE]      = "Icelake",
```

Comment:
shall we also add sapphire rapids?

Reply:
`UCS_CPU_MODEL_INTEL_SAPPHIRERAPIDS` hasn't been added to master branch yet.
Should we add Sapphire Rapids in this PR? Or is it better to cherry-pick the commit from 1.15.x branch after merging https://github.com/openucx/ucx/pull/9318?